src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/MethodeLinksBodyProcessor.java

```diff
@@ -71,51 +78,40 @@ public MethodeLinksBodyProcessor(ResilientClient documentStoreApiClient, URI uri
     @Override
     public String process(String body, BodyProcessingContext bodyProcessingContext) throws BodyProcessingException {
         if (body != null && !body.trim().isEmpty()) {
```

Comment:
And it would make this nicer if you inverted the if. So return immeditely if body == null or body.trim.isempty.
At the `if (bodyProcessingContext instanceof TransactionIdBodyProcessingContext)` as well.